### PR TITLE
Add trigger tags to root span and integration spans

### PR DESCRIPTION
**Pull request by kobalski**

### CODE REVIEW

File: src/plugins/integrations/MySQL2Integration.ts [hunk 1]
```diff
@@ -56,6 +57,10 @@ class MySQL2Integration implements Integration {
           [DBTags.DB_HOST]: this.config.host,
           [DBTags.DB_PORT]: this.config.port,
           [DBTags.DB_TYPE]: DBTypes.MYSQL,
+          [SpanTags.TOPOLOGY_VERTEX]: true,
+          [SpanTags.TRIGGER_DOMAIN_NAME]: LAMBDA_APPLICATION_DOMAIN_NAME,
+          [SpanTags.TRIGGER_CLASS_NAME]: LAMBDA_APPLICATION_CLASS_NAME,
+          [SpanTags.TRIGGER_OPERATION_NAMES]: [this.config.host],
```
Comment (serkan-ozal) [inline src/plugins/integrations/MySQL2Integration.ts]: this must be `config.database` to be consistent with the span name.

File: src/plugins/integrations/HttpIntegration.ts [hunk 2]
```diff
@@ -81,6 +82,10 @@ class HttpIntegration implements Integration {
           [HttpTags.HTTP_PATH]: path.split('?')[0],
           [HttpTags.HTTP_URL]: host + path,
           [HttpTags.QUERY_PARAMS]: queryParams,
+          [SpanTags.TOPOLOGY_VERTEX]: true,
+          [SpanTags.TRIGGER_DOMAIN_NAME]: LAMBDA_APPLICATION_DOMAIN_NAME,
+          [SpanTags.TRIGGER_CLASS_NAME]: LAMBDA_APPLICATION_CLASS_NAME,
+          [SpanTags.TRIGGER_OPERATION_NAMES]: [path.split('?')[0]],
```
Comment (serkan-ozal) [inline src/plugins/integrations/HttpIntegration.ts]: this must be `host + path` to be consistent with the span name.

Comment (kobalski) [reply src/plugins/integrations/HttpIntegration.ts]: Done

File: src/plugins/integrations/MySQL2Integration.ts (hunk 1 above)
Comment (kobalski) [reply src/plugins/integrations/MySQL2Integration.ts]: Done

**Pull request merged**